src/main/java/edit_public_profile_usecase/editPublicProfileDsRequestModel.java

```diff
@@ -0,0 +1,17 @@
+package edit_public_profile_usecase;
+
```

Comment:
Since public profile is an attribute of User, with how the User Serialization works I believe you should be able to use the same Gateway and DS Request Model as Leo does. This is because to save the changes to the user profile we need to re-save the entire hashmap of user objects, after you edit the group profile attribute of the user.